src/evict.c

```diff
@@ -545,7 +545,7 @@ int performEvictions(void) {
 
     int keys_freed = 0;
     size_t mem_reported, mem_tofree;
-    long long mem_freed; /* May be negative */
+    long long mem_freed = 0; /* May be negative */
```

Comment:
but it is initialized to 0 in line 566, 
if need we also need to delete line 566 at the same time.
i'm not sure we need to bother it, although looks werid.

btw: the reason for it being initialized on a separate line: https://github.com/redis/redis/commit/4a140d320f7a8e1a63f9c3ca588f61f21838f3ac